charts/fission-all/templates/canary-config/svc.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.canaryDeployment.enabled }}
```

Comment:
No need to create a service as we don't expose HTTP port. We can remove this file.